src/controllers/levelcontroller.js

```diff
@@ -41,6 +41,8 @@ class LevelController extends Controller
             showConfirmButton: false
         });
 
+        (new FactoryController)._updateFactory();
```

Comment:
This can also be done in `simulationcontroller` I think, after the `levelcontroller` has fired.